aio/tools/examples/shared/boilerplate/common/src/styles.css

```
@@ -43,21 +51,16 @@ nav a {
   margin-right: 10px;
   margin-top: 10px;
   display: inline-block;
-  background-color: #eee;
+  background-color: #e8e8e8;
+  color: #3d3d3d;
   border-radius: 4px;
 }
-nav a:visited, a:link {
-  color: #607D8B;
-}
+
 nav a:hover {
-  color: #039be5;
-  background-color: #CFD8DC;
-}
-nav a.active {
-  color: #039be5;
+  color: white;
+  background-color:  #42545C;
 }
-
-/* everywhere else */
-* {
-  font-family: Arial, Helvetica, sans-serif;
+nav a:active {
```

I believe the change from `a.active` to `a:active` was not intentional. This should probably be:

```suggestion
nav a.active {
```

NOTE:
- `a.active` will match the `a` element that has the `.active` class, which is many examples is associated with the `a` corresponding to the currently active route.
- `a:active` will match the `a` element that is being clicked - i.e. it will only match the element for as long as the click operation is ongoing. See also [`:active` on MDN][1].

[1]: https://developer.mozilla.org/en-US/docs/Web/CSS/:active